modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1193 @@
++
+This new default presents for both new comments and replies to existing comments.
+
+2. This default label can now be translated.
+
+As is usual in {productname}, translation of this string is done wth `addI18n()`:
+
+[source,js]
+----
+tinymce.addI18n('<optional-IETF-language-code>', {
+   'Comment': '<translation>'
+});
+----
+
+NOTE: When a comment is being edited, the default button, which writes the changes back and closes the Edit Comment dialog, is still labeled *Save*.
+
+
+==== The `change` event was not fired when deleting all conversations
+
+Previously, when all comments were deleted, existing undo levels were re-written but no new undo level was created. As a consequence no `change` event was fired.
+
+This resulted in applications that relied on {productname} reporting its current state incorrectly assuming the now-deleted comments still existed.
+
+This update corrects this. In Comments 3.1.0, deleting any or all comments fires a `change` event. And a `changed` message is, consequently, printed to the console.
+
+
+==== An additional undo level was incorrectly created when the Comments plugin was loaded at {productname} initialization
+
+Previously, when the Comments plugin loaded, a bug in the plugin caused {productname} to put the comment model data into a first undo level.
+
+This caused {productname} to present as if an undo option was available when there was no action to undo (for example, the Undo button would enable if all that had changed as the insertion point moving).
+
+This update fixes this bug. When the Comments plugin loads, the comment model data is no longer placed into an undo level and no Undo actions will present as available until there are actual changes available to undo.
+
+
+==== Opening the comments sidebar via an editor command did not respect the `skip_focus` argument
+
+If the Comments sidebar was opened with an editor command and `skip_focus` was set to true — `editor.execCommand("ToggleSidebar", false, "showcomments", { skip_focus: true });` — the sidebar still stole focus.
+
+This update corrects this: `skip_focus: true` is honored, as expected, and the sidebar does not steal focus.
+
+
+==== Comments did not blocking the Save UI while saving the data in callback mode
+
+When Comments was working in callback mode, it was possible to send multiple `save` actions from the Comment text-entry UI.
+
+Save actions are asyncrhonous, so this made it possible to generate multiple saves of the same comment.
+
+This update disables the *Save* UI when a comment is being saved, ensuring only one copy of a comment is saved at a time.
+
+
+==== A selected `hr` block could be commented on when it should not have been possible
+
+A new feature of the Comments plugin is support for annotating many block-level elements directly, using the editor annotator API (see above).
+
+A bug in this new feature, however, made it possible to, at least apparently, add a comment to an `+<hr>` block.
```

Comment:
I don't believe this is correct, but @ltrouton can correct me if I'm wrong. This was not a bug in the new feature, it was an existing bug that happened before this feature was ever implemented.

Reply:
Edited to put the blame where it belongs: on a previously undetected bug.